src/main/java/com/esri/arcgisruntime/toolkit/FeatureTemplatePicker.java

```diff
@@ -16,341 +16,221 @@
    *
    * @return the property
-   * @since 100.6.0
-   */
-  public SimpleBooleanProperty disableIfCannotAddFeatureLayersProperty() {
-    return disableIfCannotAddFeatureLayersProperty;
-  }
-
-  /**
-   * Sets the value of the {@link #disableIfCannotAddFeatureLayersProperty()}.
-   *
-   * @param disable true to disable adding if the feature layer does not support it, false otherwise
-   */
-  public void setDisableIfCannotAddFeatureLayers(boolean disable) {
-    disableIfCannotAddFeatureLayersProperty.set(disable);
-  }
-
-  /**
-   * Gets the value of the {@link #disableIfCannotAddFeatureLayersProperty()} ()}.
-   *
-   * @return true if disabled, false otherwise
-   * @since 100.6.0
+   * @since 100.7.0
    */
-  public boolean isDisableIfCannotAddFeatureLayers() {
-    return disableIfCannotAddFeatureLayersProperty.get();
+  public ObjectProperty<FeatureTemplateItem> selectedFeatureTemplateItemProperty() {
+    return selectedFeatureTemplateItem;
   }
 
   /**
-   * Property which controls the orientation of the control. Select {@link Orientation#HORIZONTAL} if the control is
-   * wider than its height and {@link Orientation#VERTICAL} if it is taller than it is wide. By default the orientation
-   * is {@code VERTICAL}.
+   * Gets the value of the {@link #selectedFeatureTemplateItemProperty()}.
    *
-   * @return the property
-   * @since 100.6.0
+   * @return the selected template or null if there is no selection
+   * @since 100.7.0
    */
-  public SimpleObjectProperty<Orientation> orientationProperty() {
-    return orientationProperty;
+  public FeatureTemplateItem getSelectedFeatureTemplateItem() {
+    return selectedFeatureTemplateItem.get();
   }
 
-  /**
-   * Sets the value of the {@link #orientationProperty()}.
-   *
-   * @param orientation the orientation
-   * @since 100.6.0
-   */
-  public void setOrientation(Orientation orientation) {
-    orientationProperty.set(orientation);
+  public void setSelectedFeatureTemplateItem(FeatureTemplateItem selectedFeatureTemplateItem) {
+    this.selectedFeatureTemplateItem.set(selectedFeatureTemplateItem);
   }
 
   /**
-   * Gets the value of the {@link #orientationProperty()}.
+   * Get the symbol size used for the feature template symbol swatches.
    *
-   * @return the orientatoin
-   * @since 100.6.0
+   * @return symbol size
+   * @since 100.7.0
    */
-  public Orientation getOrientation() {
-    return orientationProperty.get();
+  public int getSymbolSize() {
+    return symbolSize.get();
   }
 
   /**
-   * Clears any selection.
+   * Symbol size used for the feature template symbol swatches. Defaults to 20.
    *
-   * @since 100.6.0
+   * @return symbol size property
+   * @since 100.7.0
    */
-  public void clearSelection() {
-    selectedTemplateProperty.set(null);
-  }
-
-  @Override
-  protected Skin<?> createDefaultSkin() {
-    return new FeatureTemplatePickerSkin(this);
+  public IntegerProperty symbolSizeProperty() {
```

Comment:
It would be nice to have a concrete use case (example of a polyline swatch which looks better wider). That use case is also swatch-specific whereas this property applies to every swatch. I would guess that the most common use case is to have the same width & height.

Reply:
It is asked for in https://devtopia.esri.com/runtime/common-toolkit/wiki/Template-Picker
>At design time, the developer has configuration options:
>
> - to turn the names and/or descriptions on and off.
> - for the height/width/background color of the swatches.

I'm not sure what it means by design time though.

Reply:
But where is the use case that put it in that list?

The item and group names can be hidden via css. The background color of the toggle button could be changed via css too (on the image view).

This does not feel like min-ship to me.